### ISSUE TITLE
How to get Dashboard URL for a specific invocation?

### ISSUE THREAD

**Issue by patricktyndall** [serverless/serverless#12173]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

Hi. Prior to this week's Dashboard release, `context.serverlessSdk.getDashboardUrl()` could be used to get a link to the dashboard for an invocation. Other identifiers were available as well (like `getTransactionId()`). 

These identifiers are extremely important for correlating errors in lambda to the associated serverless logs. We have now lost this capability. Now that dashboard URLs have changed, these links no longer work (I can't find a similar function in SDK V3).

How can we use the SDK to get an identifier or URL for the trace of a specific invocation?

### Service configuration (serverless.yml) content

```yaml
n/a
```

### Command name and used flags

n/a

### Command output

```shell
n/a
```

### Environment information

```shell
n/a
```

**Comment by Alex-from-Ukraine** [comment]
I agree, it was very convenient to be able to save a link when calling a lambda, so that later you could quickly find the corresponding log. I also couldn't find a similar feature in either @serverless/sdk or @serverless/aws-lambda-sdk. Could you tell me please how can I get a similar link on the dashboard's page with the current Lamba trace?

**Comment by vadymhimself** [comment]
+1

**Comment by Alex-from-Ukraine** [comment]
![image](https://github.com/serverless/serverless/assets/132647835/fa9f5fd9-e197-42a5-a818-939de57978cf)

**Comment by vadymhimself** [comment]
+1
